cdap-proto/src/main/java/co/cask/cdap/proto/ProgramTypes.java

```diff
@@ -33,16 +33,16 @@
  */
 public class ProgramTypes {
 
-  private static final Map<Class<? extends ProgramSpecification>, ProgramType> specClassToProgramType =
-    ImmutableMap.<Class<? extends ProgramSpecification>, ProgramType>builder()
-      .put(FlowSpecification.class, ProgramType.FLOW)
-      .put(MapReduceSpecification.class, ProgramType.MAPREDUCE)
-      .put(SparkSpecification.class, ProgramType.SPARK)
-      .put(WorkflowSpecification.class, ProgramType.WORKFLOW)
-      .put(WebappSpecification.class, ProgramType.WEBAPP)
-      .put(ServiceSpecification.class, ProgramType.SERVICE)
-      .put(WorkerSpecification.class, ProgramType.WORKER)
-      .build();
+  private static final Map<Class<? extends ProgramSpecification>, ProgramType> specClassToProgramType = new HashMap<>();
```

Comment:
I thought about immutability. However, seems like this one is private and never exposed as a map outside of this class, so should be ok?